integration-tests/src/jvmTest/kotlin/ai/koog/integration/tests/KotlinAIAgentWithMultipleLLMIntegrationTest.kt

```diff
@@ -473,7 +473,7 @@ class KotlinAIAgentWithMultipleLLMIntegrationTest {
 
             val compressHistoryNode by nodeLLMCompressHistory<Unit>("compress_history")
 
-            nodeStart then anthropicSubgraph then compressHistoryNode then openaiSubgraph then nodeFinish
+            applyLinearStrategy(anthropicSubgraph then compressHistoryNode then openaiSubgraph)
```

Comment:
So, basically, one side of this, is that users don't have to write `nodeStart` and `nodeFinish` all the time